### PR TITLE
Fixed cut text in Views

### PR DESCRIPTION
**Pull request by n1snt**
### Issue: 
The text cuts out when the text size increased.

### Expected behaviour:
View scales with respect to the text size.

### Views progress:
- [x] Fragment phone
- [x] Fragment Advanced timetable
- [x] Edit bookmark
- [x] Add place done button (English is ok but there seems to be an issue with German)
- [x] item_add_street

### Issues:
https://github.com/organicmaps/organicmaps/issues/4774
https://github.com/organicmaps/organicmaps/issues/4417

### CODE REVIEW

**Review by biodranik — COMMENTED** [review]
Great! What about other dialogs? Can you please show some screenshots with the largest font set in system settings?

**Comment by rtsisyk** [comment]
a screenshot please

**Comment by n1snt** [comment]
Fragment phone:
Before             |  After
:-------------------------:|:-------------------------:
image | image

Fragment advanced timetable:
Before             |  After
:-------------------------:|:-------------------------:
image | image

**Comment by n1snt** [comment]
Add place done button:
Before             |  After
:-------------------------:|:-------------------------:
image | image

**Comment by biodranik** [comment]
Thanks! Only two checkboxes left )

**Comment by n1snt** [comment]
I'm currently a bit occupied with completing my GSOC proposal.
Will get back to this as soon as I am done with the first version of my proposal :)

**Comment by n1snt** [comment]
Edit bookmarks:
Before             |  After
:-------------------------:|:-------------------------:
image | image

**Comment by n1snt** [comment]
Position chooser:
(The text only appears cut in german)
Before             |  After
:-------------------------:|:-------------------------:
image | image

**Review by Jean-BaptisteC — APPROVED** [review]
✅ Pixel 6 - Android 13
Tested with max police size and max police size and max UI size

❌item add_street is not fixed (in French letter `j` in `Ajouter` is cut

**Comment by n1snt** [comment]
Can you please share a screenshot?

**Comment by n1snt** [comment]
@Jean-BaptisteC just pushed a fix for it.
Can you please check if it works now?

**Comment by Jean-BaptisteC** [comment]
Thanks LGTM
Please sign your commits

File: android/res/values/dimens.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
   <dimen name="height_block_base">48dp</dimen>
   <dimen name="height_item_oneline">56dp</dimen>
   <dimen name="height_item_multiline">72dp</dimen>
-  <dimen name="height_item_edit_bookmark">72dp</dimen>
+  <dimen name="height_item_edit_bookmark">87dp</dimen>
```
Comment (biodranik) [inline android/res/values/dimens.xml:37]: Simply increasing the max height won't work for even larger fonts. The height should be dynamical.

Comment (n1snt) [reply android/res/values/dimens.xml:37]: I have successfully made it dynamic now.
Here are some screenshots:
Edit bookmarks:
Before             |  After
:-------------------------:|:-------------------------:
image | image

**Comment by n1snt** [comment]
@Jean-BaptisteC Signed the commits ✅

File: android/res/layout/edit_bookmark_common.xml [hunk 2]
```diff
@@ -55,22 +54,23 @@
       android:background="?attr/selectableItemBackground"
       android:clickable="true"
       android:paddingTop="@dimen/margin_quarter_plus"
-      android:paddingBottom="@dimen/margin_half"
```
Comment (biodranik) [inline android/res/layout/edit_bookmark_common.xml:58]: Is this identifier used somewhere else? If not, please clean it up.

File: android/res/layout/fragment_phone.xml [hunk 3]
```diff
@@ -25,7 +25,7 @@
     <TextView
         android:id="@+id/tv__append_phone"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/height_block_base"
```
Comment (biodranik) [inline android/res/layout/fragment_phone.xml:28]: Is this identifier used somewhere else? If not, please clean it up.

File: android/res/layout/item_add_street.xml [hunk 4]
```diff
@@ -3,7 +3,7 @@
   xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
   android:layout_width="match_parent"
-  android:layout_height="@dimen/height_item_oneline"
```
Comment (biodranik) [inline android/res/layout/item_add_street.xml:6]: Is this identifier used somewhere else? If not, please clean it up.

File: android/res/layout/edit_bookmark_common.xml [hunk 5]
```diff
@@ -11,7 +11,7 @@
   <LinearLayout
     android:id="@+id/ll__bookmark_name"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/height_item_edit_bookmark"
```
[the remark this answers is not in the document]
Comment (n1snt) [reply android/res/layout/edit_bookmark_common.xml:14]: This identifier is in use in item_bookmark_category.xml

File: android/res/layout/edit_bookmark_common.xml (hunk 2 above)
Comment (n1snt) [reply android/res/layout/edit_bookmark_common.xml:58]: This is used in a lot of other places
image

File: android/res/layout/fragment_phone.xml (hunk 3 above)
Comment (n1snt) [reply android/res/layout/fragment_phone.xml:28]: This is also used in a lot of other places
image

File: android/res/layout/item_add_street.xml (hunk 4 above)
Comment (n1snt) [reply android/res/layout/item_add_street.xml:6]: Same for this 
image

**Comment by biodranik** [comment]
Good, thanks! @Jean-BaptisteC is everything fixed now? Can we merge it?

**Comment by Jean-BaptisteC** [comment]
> Bien, merci !@Jean-BaptisteCtout est réglé maintenant ? Peut-on le fusionner ?

Yes

**Review by vng — APPROVED** [review]

**Pull request merged**